src/rgw/rgw_redis_scripts.h

```diff
@@ -0,0 +1,236 @@
+#pragma once
+
+#include <string>
```

Comment:
please mak esure that there is a generated comment here saying this is an autogenrated file

Reply:
I can add the comment to the template file that is used to generate this header file. 
Btw, what is a good location to keep the template file itself? 

Is the current location good?